installVm-linux.sh

```diff
@@ -0,0 +1,10 @@
+#Check if a vm already is downloaded, if so go ahead, if not download vm 
+
+if ! which multipass > /dev/null 2>&1; 
+then
+  echo 'installing multipass'
+  sudo snap install multipass
```

Comment:
If you are on a mac, you would be installing Multipass on your local mac. So you will need this to work on mac.

Also, your logic is backwards than what we cover in the sessions. The IF statements we use only "do work" in the else, so you are checking "it should be true that multipass is installed" and so it's more like :

```
if (which multipass)
then
  echo "multipass is already installed"
else
  echo "installing mulitpass"
  # command to install mulitpass
fi
```